paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -152,10 +155,7 @@ internal class DefaultFlowController @Inject internal constructor(
         }
 
     init {
-        val paymentLauncherActivityResultLauncher = activityResultRegistryOwner.register(
-            PaymentLauncherContract(),
-            ::onInternalPaymentResult
-        )
+        intentConfirmationHandler.register("FlowController", activityResultRegistryOwner, lifecycleOwner)
```

Comment:
Why does flow controller need this, and the others don't?

Reply:
Discussed in Slack, we resolved this with a wrapper that lets us use `ActivityResultCaller` in `FlowController`.